src/cgppindel.sh

```
@@ -17,6 +17,7 @@ main() {
     echo "Value of filter: '$filter'"
     echo "Value of tumour: '$tumour'"
     echo "Value of normal: '$normal'"
+    echo "Value of exclude: '$exclude'"
```

Shellcheck (0.11.0)

[warning] exclude is referenced but not assigned.

(SC2154)